README.adoc

```diff
@@ -7,7 +7,10 @@ image:https://github.com/{repository-owner}/{repository-name}/actions/workflows/
 image:https://github.com/{repository-owner}/{repository-name}/actions/workflows/compile-examples.yml/badge.svg["Compile Examples status", link="https://github.com/{repository-owner}/{repository-name}/actions/workflows/compile-examples.yml"]
 image:https://github.com/{repository-owner}/{repository-name}/actions/workflows/spell-check.yml/badge.svg["Spell Check status", link="https://github.com/{repository-owner}/{repository-name}/actions/workflows/spell-check.yml"]
 
-This library allows the control and configuration of the BQ24195 PMIC used on various Arduino boards.
+This library allows the control and configuration of the BQ24195 PMIC used on various Arduino boards including:
```

Comment:
```suggestion
This library allows the control and configuration of the BQ24195 PMIC used on various Arduino boards including:

```
```suggestion

- All MKR boards (apart from MKR 1000, Zero, WAN 1300 and Vidor 4000)
```
A leading blank line is required to cause the Asciidoc list formatting to be recognized.

Without the blank line, it is rendered like this:

![image](https://user-images.githubusercontent.com/8572152/184684521-9c9bdcb2-a599-4769-ba73-3a120c5edef1.png)


With the blank line, the rendering is as expected:

![image](https://user-images.githubusercontent.com/8572152/184684695-49307359-00dd-448b-ad71-023d59aa9ffd.png)

Reply:
@aliphys the problem with the incorrect Asciidoc list markup is still present. Check the rendering here:

https://github.com/aliphys/Arduino_BQ24195#readme

Information about the Asciidoc list syntax here:

https://docs.asciidoctor.org/asciidoc/latest/lists/unordered/

Reply:
I looked at the document you shared, and adjusted the unordered list formatting accordingly. It should be correct now.
![image](https://user-images.githubusercontent.com/75624145/184847644-079b5ff4-652f-412c-866d-ac0bd7d34872.png)